src/env/index.ts

```diff
@@ -0,0 +1,254 @@
+import { Cell, Direction, GameActions, GameState, Player } from "./types"
+
+export const GridWidth = 42 // columns
```

Comment:
@aronluigi according to requirements `GridWidth` & `GridHeight` should be calculated from the screen size.

Reply:
I misunderstood the requirements regarding this point.